app/src/ui/lib/list/list.tsx

```diff
@@ -524,11 +524,14 @@ export class List extends React.Component<IListProps, IListState> {
       this.props.onRowKeyDown(rowIndex, event)
     }
 
+    const isSpecialShortcut = __DARWIN__ ? event.metaKey : event.ctrlKey
```

Comment:
Instead of checking whether the keyboard shortcut _isn't_ `CmdOrCtrl+Enter` we should check that it is 'Enter' or 'Space' without any modifiers. The first approach implies that the List component is aware of a specific keyboard shortcut used globally and avoids it whereas the second approach makes the List component specifically look for modifier-less `Enter` or `Space`

I'm working on a PR which will allow us to use Electron-style accelerators but for now checking that no modifiers are set should suffice.

```suggestion
    const hasModifier =  event.altKey || event.ctrlKey || event.metaKey || event.shiftKey
```